conjure-java-server-verifier/src/test/java/com/palantir/conjure/java/verification/server/undertest/UndertowServerUnderTestExtension.java

```diff
@@ -37,8 +36,7 @@
 
     @Override
     public void beforeAll(ExtensionContext _context) {
-        UndertowAutoDeserializeService autoDeserialize =
-                Reflection.newProxy(UndertowAutoDeserializeService.class, new EchoResourceInvocationHandler());
+        UndertowAutoDeserializeService autoDeserialize = new EchoUndertowAutoDeserializeService();
```

Comment:
Using a proxy that calls a default method on a non-private-accessible interface is quite tricky as you have to use a different workaround depending on your jvm version ([reference](Calling a Default Method on a Non-Private-Accessible Interface)). Based on that, I manually generated the implementation that mirrors the behavior the previously used proxy.